docs/external_system.md

```diff
@@ -46,13 +46,13 @@ use externally-accessible API endpoints exposed by CSM.
    $ . ${SAT_VENV_PATH}/bin/activate
    ```
 
-1. Clone the SAT source code.
+1. (`venv`) Clone the SAT source code.
```

Comment:
This one is not something that I see reflected in the docs-csm command prompt conventions, but I think it's sort of similar to a chroot [as documented here](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/introduction/documentation_conventions.md#command-prompts-inside-new-shells). It's not exactly a new shell, but it does need to be run in a certain environment which is set up by running the previous command, `. ${SAT_VENV_PATH}/bin/activate`.

I think the way you've done it here is good, but perhaps we should mention it in our command prompt conventions in docs/about_sat/introduction.md

Reply:
One possibility:

```
(`(venv) user@hostname>`)
```

Reply:
Updated in next commit.